src/StackExchange.Redis/ResultProcessor.cs

```diff
@@ -1262,6 +1265,22 @@ protected override bool SetResultCore(PhysicalConnection connection, Message mes
             }
         }
 
+        private sealed class BooleanArrayProcessor : ResultProcessor<Boolean[]>
```

Comment:
Use C# types please:
```suggestion
        private sealed class BooleanArrayProcessor : ResultProcessor<bool[]>
```